glfw/buildfile

```diff
@@ -1,5 +1,5 @@
-./: {*/ -build/ -upstream/} doc{README.md} legal{LICENSE.md} manifest
+./: {src/} doc{README.md} legal{LICENSE.md} manifest
```

Comment:
Why the change here? It shouldn't be necessary; it's probably best to keep it as it was traditionally (?)

Reply:
The correct thing should be `{*/ -build/ -deps/}`